app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,76 @@
+class FoodsController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_food, only: %i[show destroy]
+  # before_action :authorize_delete, only: [:destroy]
```

Comment:
- [ ] Here as well, I would recommend removing the commented code to make the codebase cleaner.